cpp/tests/test_abm_person.cpp

```diff
@@ -24,56 +24,40 @@ TEST(TestPerson, init)
     using testing::Return;
 
     auto location = mio::abm::Location(mio::abm::LocationType::Work, 0);
-    //setup rng mock so the time_until_carrier is 1.
-    ScopedMockDistribution<testing::StrictMock<MockDistribution<mio::UniformIntDistribution<int>>>>
-        mock_uniform_int_dist;
-    EXPECT_CALL(mock_uniform_int_dist.get_mock(), invoke).Times(testing::AtLeast(1)).WillRepeatedly(Return(0));
+    auto t        = mio::abm::TimePoint(0);
+    auto person   = create_person_simple(location, mio::abm::AgeGroup::Age60to79, mio::abm::InfectionState::Exposed);
 
-    auto person = mio::abm::Person(location, mio::abm::InfectionState::Exposed, mio::abm::AgeGroup::Age60to79, {});
-    location.add_person(person);
-
-    ASSERT_EQ(person.get_infection_state(), mio::abm::InfectionState::Exposed);
-    ASSERT_EQ(person.get_location_id().index, location.get_index());
-    ASSERT_EQ(person.get_location_id().type, location.get_type());
+    ASSERT_EQ(person.get_infection_state(t), mio::abm::InfectionState::Exposed);
```

Comment:
I think this test mostly tests `create_person_simple`, not the constructor of `Person`, as intended.